web/src/components/PageHeader.tsx

```diff
@@ -60,15 +60,13 @@ export const PageHeader: React.FunctionComponent<Props> = ({ title }: Props) =>
           </a>
           <div className={classes.title}>{title}</div>
         </div>
-        <a
-          className={classes.contact}
-          href={FIDER_LINK}
-          target="_blank"
-          rel="noopener noreferrer"
+        <div
           id="contact-link"
+          className={classes.contact}
+          onClick={() => window.open('mailto:bcws.predictiveservice@gov.bc.ca')}
```

Comment:
I think we should add a subject (maybe even a body?) - Something like "Predictive Services Unit - Website" - it's maybe too much trouble to add in "Percentile Calculator" or "Morecast" depending on where you are?
```suggestion
          onClick={() => window.open('mailto:bcws.predictiveservices@gov.bc.ca?subject=Predictive Services Unit - Website')}
```

Reply:
Okay sounds good I will add the broader subject

Reply:
Change of mind. Turns out it's fairly easy to be more specific